src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -181,7 +181,9 @@ private function parseImports(array $content, $file)
         $defaultDirectory = dirname($file);
         foreach ($content['imports'] as $import) {
             if (!is_array($import)) {
-                throw new InvalidArgumentException(sprintf('The values in the "imports" key should be arrays in %s. Check your YAML syntax.', $file));
+                $import = array('resource' => $import);
```

Comment:
only strings should be allowed to be rewritten that way, not any non-array value